multimedia/VLC/Portfile

```diff
@@ -247,23 +206,17 @@ if {(${subport} eq ${name}) || (${subport} eq "lib${name}")} {
                 ${worksrcpath}/modules/codec/avcodec/avcommon_compat.h
         }
     }
-    post-configure {
-        system -W ${worksrcpath} "patch -Np0 -i ${filespath}/patch-static_assert.diff"
-    }
 
-    use_autoreconf      yes
+    # do autoreconf only the first time around
+    use_autoreconf      [expr ![file exists ${worksrcpath}/config.status]]
```

Comment:
The configure phase (which runs autoreconf) is only executed once unless it failed before (or the state file was manually edited). This is useful for developing the Portfile, but not for users. Please revert this block.